contracts/tonomy/include/tonomy/tonomy.hpp

```diff
@@ -253,7 +253,7 @@ namespace tonomysystem
       // Create an instance of the table that can is initalized in the constructor
```

Comment:
If you are going to change this then I would like you to change it to something else. I think we are going to need a single turntable which contains all kinds of network settings. Therefore I would like to have one settings object where we put all this stuff in

```c++
struct [[eosio::table]] network_settings
      {
         double ram_fee;                      // RAM fee fraction (0.01 = 1% fee)
         double ram_price;                    // RAM price (bytes per token)
	@@ -262,9 +262,9 @@ namespace tonomysystem
         uint64_t total_cpu_weight_allocated; // Total allocated (CPU weight)
         uint64_t total_net_weight_allocated; // Total allocated (NET weight)

         EOSLIB_SERIALIZE(resrc_config, (ram_price)(total_ram_available)(total_ram_used)(total_cpu_weight_allocated)(total_net_weight_allocated)(ram_fee))
      };
      typedef eosio::singleton<"netsettings"_n, network_settings> network_settings_table;
```

Reply:
any other name suggestion as i think this name also have more than 13 character as i changed name due to more characters